src/download.c

```diff
@@ -164,6 +164,7 @@ static void free_curl_list_data(void *data)
 {
 	struct file *file = (struct file *)data;
 	CURL *curl = file->curl;
+	swupd_download_file_complete(CURLE_OK, file);
```

Comment:
The codebase is not consistent here, but I would like to see an explicit (void) cast to indicate an unused return value.